src/wasm-lib/kcl/src/std/revolve.rs

```diff
@@ -255,10 +247,10 @@ async fn inner_revolve(
     args: Args,
 ) -> Result<Box<ExtrudeGroup>, KclError> {
     if let Some(angle) = data.angle {
-        // Return an error if the angle is less than -360 or greater than 360.
-        if !(-360.0..=360.0).contains(&angle) {
+        // Return an error if the angle is less than -360 or greater than 360 and cannot be 0.0
+        if !(-360.0..=360.0).contains(&angle) || angle == 0.0 {
```

Comment:
Should this do an epsilon comparison? I.e. also error if it's within epsilon of zero? As for what the value of epsilon should be... I did a quick search of modeling-app, and I didn't find anything that looks like what we want. Might need to see what the engine needs.

Reply:
we can validate this with the same library we use in settings https://docs.rs/validator/latest/validator/ with a derive, makews it easier and it integrates into json scehma so the docs will auto get it, its win win

Reply:
happy to pair if you need help

Reply:
I talked to Adam about this in Rust Club, we ended up opting for using the same pattern in `inner_revolve`. My first implementation of the `validator` did not work as expected. Also I think when we tried doing the `validator` method in rust club it had an issue? Then we thought about making the `Option<f64>` into an `Option<RevolveAngle>` struct that would do the verification?

Yeah, I'll pair if you want!

Reply:
okay yeah if we use validator here we get weirder errors but we can at least add the range to the schemars like so: https://github.com/KittyCAD/modeling-app/commit/23cd16a2329199db43120c9d17f82c1154ebe418#diff-b4e1769ab46a04e0d1bde955e84efb6360ca5d8165dcbe3f9261f572021fd0e3

Reply:
I added it to arc too, we dont currently use it for the markdown docs but at least its in the generated schema